src/main/java/com/mercuryCyclists/Inventory/service/ProductService.java

```diff
@@ -93,15 +93,27 @@ public boolean addPart(Long productId, Part part) {
         return true;
     }
 
+    /**
+     * Update part helper method
+     * @param product
+     * @param part
+     * @return If given product contains given part return true else false
+     */
+    private boolean updatePartHelper(Product product, Part part) {
+        return product.getSet().contains(part);
+    }
```

Comment:
This shouldn't be needed. 

As the product has a set, when you add the ID if there is a duplicate java will not add it.
The only benefit of this method is that we can throw an error indicating that it is already included (returned null), which is not being done.

Also, in general you could come up with a better naming convention for the method if it was needed.